### PR TITLE
add cython

### PR DESCRIPTION
**Pull request by CJ-Wright**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (mariusvniekerk) [inline recipe/meta.yaml]: can we switch to pip here

Comment (CJ-Wright) [reply recipe/meta.yaml]: Most likely, I wasn't going to because I wanted to make this PR minimally invasive. If a CI passes I'll put in pip.

Comment (jakirkham) [reply recipe/meta.yaml]: This means we need `{{ compiler("c") }}` as well, right?

Comment (CJ-Wright) [reply recipe/meta.yaml]: I think it is already in there.

**Comment by sodre** [comment]
@CJ-Wright @mariusvniekerk thanks for submitting the PR and reviewing it. I am just waiting for the Windows build to complete.

**Pull request merged**